python/lsst/pipe/analysis/utils.py

```diff
@@ -410,6 +508,75 @@ def __call__(self, catalog):
         return np.array(e2Resids)*self.unitScale
 
 
+class RhoStatistics(object):
+    """Functor to compute Rho statistics (Rowe (2010), Jarvis et al., (2016))
+    from a given star catalog and PSF model
+
+    Parameters
+    ----------
+
+    column : `str`
+        The name of the shape measurement algorithm (SdssShape or HsmRegauss).
+    psfColumn: `str`
+        The name used for PSF shape measurements from the same algorithm.
+    **kwargs
+        Additional keyword arguments passed to treecorr
+
```

Comment:
Possibly a link to the treecorr docs so you can see what these could be, up to you though if you think that would be helpful.